### ISSUE TITLE
Incorrect GrFN wiring for PETPT example

### ISSUE THREAD

**Issue by adarshp** [ml4ai/delphi#184]
The GrFN wiring for PETPT seems to be incorrect - based on these lines:

```fortran
      IF (XHLAI .LE. 0.0) THEN
        ALBEDO = MSALB
      ELSE
        ALBEDO = 0.23-(0.23-MSALB)*EXP(-0.75*XHLAI)
      ENDIF
```

The wiring below is produced (I've marked the nodes and edges that should be removed).

<img width="721" alt="screen shot 2019-02-21 at 12 26 23 pm" src="https://user-images.githubusercontent.com/653549/53196918-5feaaf00-35d6-11e9-90d0-a4a826c81081.png">

@pratikbhd If this is a relatively easy fix, please prioritize it as @cl4yton will likely be demoing this particular example on Monday at the WM PI meeting.

**Comment by pratikbhd** [comment]
![image](https://user-images.githubusercontent.com/19292952/53198288-46973200-35d9-11e9-96d6-bb982a8f7807.png)

@adarshp Just to confirm, this entails the link highlighted in blue to be removed as well, right?

**Comment by adarshp** [comment]
Yes, that's right - thanks for catching that!

**Comment by adarshp** [comment]
Closed by #188
